src/main/java/org/apache/commons/lang3/reflect/ConstructorUtils.java

```diff
@@ -105,6 +107,7 @@ public ConstructorUtils() {
      * @throws InstantiationException if an error occurs on instantiation
      * @see Constructor#newInstance
      */
+    @SuppressWarnings({"argument.type.incompatible", "value:assignment.type.incompatible"}) // #1 isVarArgs => args has at least one element
```

Comment:
This is an example of a place where the suppress warning key should be prefixed with the checker to which it should apply.